src/game-of-life/implementing.md

```diff
@@ -390,6 +390,109 @@ Make sure your development server is still running (run `npm run start` inside
 
 [![Screenshot of the Game of Life implementation with text rendering](../images/game-of-life/initial-game-of-life-pre.png)](../images/game-of-life/initial-game-of-life-pre.png)
 
+## Testing our Game of Life
+
+Now that we have our Rust implementation of the Game of Life rendering in the browser with JavaScript, let's talk about testing our Rust compiled to WebAssembly functions.
+
+We are going to test our `tick` function to make sure that it gives us the output that we expect.
+First, we'll want to create some setter functions inside our `impl Universe` in the `wasm_game_of_life/src/lib.rs` file. We are going to create a `set_width` and a `set_height` function so we can create `Universe`s of different sizes.
+
+```rust
+#[wasm_bindgen]
+impl Universe { 
+    // ...
+
+    pub fn set_width(mut self, width: u32) -> Self {
+        self.width = width;
+        // Reinitialize cells with new width and the universe's height.
+        // Set all the cells to dead.
```

Comment:
Taking ownership of `self` and returning a "new" `Self` is very pseudo-functionally -- your Haskell is showing ;) But this isn't generally the most idiomatic Rust style. Taking `&mut self` is equivalent, and a bit more natural.

Additional nitpick: Since these are public functions, and this behavior is something that a caller would want to know, I think this should be a doc comment rather than an internal comment:

```rust
/// Set the width of the universe.
///
/// Resets all cells to the dead state.
pub fn set_width(&mut self) {
    ...
}
```